aplib.net-demo/Assets/Scripts/Entities/Player/MouseLock.cs

```diff
@@ -4,6 +4,11 @@ public class MouseLock : MonoBehaviour
 {
     private bool _showMouse = false;
 
+    /// <summary>
+    /// Manages all the UI canvases.
+    /// </summary>
+    public CanvasManager CanvasManager;
```

Comment:
Wordt deze alleen in deze file gebruikt? Dan kan die namelijk private gemaakt worden.